Sources/Controllers/MessagesViewController.swift

```diff
@@ -40,14 +40,14 @@ UICollectionViewDelegateFlowLayout, UICollectionViewDataSource {
     /// last item whenever the `InputTextView` begins editing.
     ///
     /// The default value of this property is `false`.
-    /// NOTE: This calls scrollToLastItem where as the below flag calls scrollToBottom - check methods for differences
+    /// NOTE: This is related to `scrollToLastItem` whereas the below flag is related to `scrollToBottom `- check each function for differences
```

Comment:
Can you apply just these comment fixes to develop in a separate PR? I'd like to have them in place now rather than later.